bin/run

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env node
 
-const oclif = require('@oclif/core')
+// Add this line to mute oclif deprecation messages
+process.env.OCLIF_HIDE_DEPRECATION = "1";
```

Comment:
😮  how'd you find this env var

Reply:
I asked chatGPT 😅 , and it seems that Oclif's official documentation or release notes might mention this environment variable as a way to suppress deprecation warnings.

Reply:
i'm not sure about this env var, i cant find any documentation anywhere. an alternative might be this https://stackoverflow.com/questions/55778283/how-to-disable-warnings-when-node-is-launched-via-a-global-shell-script

Reply:
Do u want to keep both?

Reply:
I haven't :/

Reply:
let's remove this for now until we find a way to repro. the other change looks good we can merge that